BSP file - device-tree

As it was reported on the adapteva forum and our team confirmed. The parallella
board doesnt boot if the UART cable is not connected. This patch fixes the
issues by removing the UART as the console.

Signed-off-by: Theodor Gherzan <theodor@resin.io>

--- parallella/parallella-mmc-boot.dts	2015-05-04 13:05:30.361127391 +0200
+++ parallella/parallella-mmc-boot.dts	2015-05-04 13:10:09.942899354 +0200
@@ -3,6 +3,7 @@
 /include/ "parallella.dtsi"
 / {
 	chosen {
-		bootargs = "console=ttyPS0,115200 root=/dev/mmcblk0p2 rw rootwait earlyprintk";
+       bootargs = "root=/dev/mmcblk0p2 rw rootwait earlyprintk";
+       linux,stdout-path = "/amba@0/uart@E0001000";
 	} ;
 } ;
